dlt/common/schema/typing.py

```diff
@@ -171,6 +171,7 @@ class TTableSchema(TypedDict, total=False):
     columns: TTableSchemaColumns
     resource: Optional[str]
     table_format: Optional[TTableFormat]
+    insert_api: Optional[Literal["streaming", "default"]]
```

Comment:
I guess it's okay that it's a global param. BigQuery is not the only service, which allows different insert approaches.

Reply:
I'd prefer not to specify it until we have more cases... please use `x-` prefix to have custom hints.